knip-reporter.ts

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
Do we need this custom reporter? If yes, maybe we could create the file in the pipeline especially given that it would be added to the root of the repository